tests/integration/Owncloud/OcisPhpSdk/DriveTest.php

```diff
@@ -696,4 +696,38 @@ public function testReceiverUpdatesDriveShareRoleNotEnoughPermission(): void
             $receivedInvitationDrive->setPermissionRole($permissionId, $role);
         }
     }
+
+    /**
+     * @throws ForbiddenException
+     * @throws InvalidResponseException
+     * @throws BadRequestException
+     * @throws UnauthorizedException
+     * @throws HttpException
+     * @throws NotFoundException
+     * @throws InternalServerErrorException
+     * @throws \Exception
+     */
+    public function testSearchResourceInsideDrive(): void
+    {
+        $this->drive->createFolder('myfolder');
+        $this->drive->uploadFile('myfolder/somefile.txt', 'some content');
+        $this->drive->uploadFile('somefile.txt', 'root content');
```

Comment:
also upload files that should not be found, to make sure the function not just returns all files